gsplat/cuda/_wrapper.py

```diff
@@ -572,6 +572,34 @@ def rasterize_to_indices_in_range(
     return out_gauss_ids, out_pixel_ids, out_camera_ids
 
 
+def compute_3D_smoothing_filter(
```

Comment:
nit: maybe use `3d` instead of `3D` in the naming of these functions to align with the naming convention in gsplat?